apriltag.c

```diff
@@ -265,29 +266,29 @@ static void quick_decode_init(apriltag_family_t *family, int maxhamming)
 
     family->impl = qd;
 
-    if (0) {
-        int longest_run = 0;
-        int run = 0;
-        int run_sum = 0;
-        int run_count = 0;
-
-        // This accounting code doesn't check the last possible run that
-        // occurs at the wrap-around. That's pretty insignificant.
-        for (int i = 0; i < qd->nentries; i++) {
-            if (qd->entries[i].rcode == UINT64_MAX) {
-                if (run > 0) {
-                    run_sum += run;
-                    run_count ++;
-                }
-                run = 0;
-            } else {
-                run ++;
-                longest_run = imax(longest_run, run);
-            }
-        }
-
-        printf("quick decode: longest run: %d, average run %.3f\n", longest_run, 1.0 * run_sum / run_count);
-    }
+    // if (0) {
+    //     int longest_run = 0;
+    //     int run = 0;
+    //     int run_sum = 0;
+    //     int run_count = 0;
+
+    //     // This accounting code doesn't check the last possible run that
+    //     // occurs at the wrap-around. That's pretty insignificant.
+    //     for (int i = 0; i < qd->nentries; i++) {
+    //         if (qd->entries[i].rcode == UINT64_MAX) {
+    //             if (run > 0) {
+    //                 run_sum += run;
+    //                 run_count ++;
+    //             }
+    //             run = 0;
+    //         } else {
+    //             run ++;
+    //             longest_run = imax(longest_run, run);
+    //         }
+    //     }
+
+    //     debug_print("quick decode: longest run: %d, average run %.3f\n", longest_run, 1.0 * run_sum / run_count);
+    // }
```

Comment:
Don't comment code. This should either be guarded by some compiler flag so it can be activated if required, or it should be deleted entirely.